vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/treegrid/it/TreeGridExpandDataRequestIT.java

```diff
@@ -17,14 +17,6 @@ public void before() {
 
         setupTreeGrid();
         findElement(By.id("LoggingDataProvider")).click();
-
```

Comment:
Reverted https://github.com/vaadin/flow-components/commit/f81f79e2c133191c660687428a1f662b4f27ef4b#diff-aa65b3cdd8b798c2156938985958fc695fac474a1cf728b0e633fcb4b4f7acaa since it's no longer valid after this update